### ISSUE TITLE
Low accuracy on trillionpairs

### ISSUE THREAD

**Issue by tranvanhoa533** [deepinsight/insightface#554]
Hello @nttstar 
I train LResNet100E-IR with emore + asian dataset on 1080Ti gpu. The accuracy on trillionpairs challenge is very low:

             m         per_batch_size    num_gpu      batch_size     identification         verification
      1     0.3               62           8              496              71%                  70.6%
      2     0.5               62           7              434              55.4%                55%
      3     0.5               180          6              1080             29.7%                27%
(In third experiment, I used mxnet-memonger to decrease memory).

I am very puzzled. Can you point out my mistake, please ? Thank you very much.

**Comment by nttstar** [comment]
Make sure the merging process is correct. It should get 80%+ easily with m=0.5 arcface loss.

**Comment by tranvanhoa533** [comment]
I downloaded glint asian and emore dataset from the [link](https://github.com/deepinsight/insightface/wiki/Dataset-Zoo)  which you shared. After that, i run dataset_merge.py to merged them.

**Comment by tranvanhoa533** [comment]
Hi @nttstar 
I did another experiment with your default parameters ( per-batch-size = 128, num-gpu = 4, r100, m=0.5 arcface loss , dataset: emore+asian) but the result on trillionpairs challenge was only 61.19% (Identification) and 59.17% (Verification). I tried to duplicate your experiment but i could not achieve your result (84%).  Did you clean the dataset when you merged them ? Did you have other techniques ? Can you share them, please ? Thank you very much !

**Comment by jeremmyzong** [comment]
> Make sure the merging process is correct. It should get 80%+ easily with m=0.5 arcface loss.

Hi @nttstar 
Did you use pretrained model to deduplicate the dataset in merging process?
Thanks!

**Comment by jeremmyzong** [comment]
Hi @tranvanhoa533 
I met the same problem, and I did experiment with bigger batch_size(per-batch-size = 128, num-gpu = 8), only get 43%(id) and 39%(verification). Have you solved it?

**Comment by tranvanhoa533** [comment]
Hi @jeremmyzong 
I still don't understand why that is. Have you solved it ?

**Comment by eguoguo321** [comment]
how to use mxnet-memonger，when I enabled that  to be true, it met error.
what else should I do?

**Comment by maywander** [comment]
Did you merge these two datasets successfully?@Talgin

**Comment by Talgin** [comment]
Hi @maywander ,
As an answer you can look this thread #256 
Thank you! :)

**Comment by maywander** [comment]
Thanks for your reply. I've solved the problem.

At 2019-10-09 17:09:48, "Talgin" <notifications@github.com> wrote:


Hi @maywander ,
As an answer you can look this thread #256
Thank you! :)
